packages/node-core/src/indexer/dictionary.service.ts

```diff
@@ -174,6 +247,7 @@ export class DictionaryService {
     let link: ApolloLink;
 
     if (this.nodeConfig.dictionaryResolver) {
+      this.supportsCursor = false;
```

Comment:
This wont work, if the dictionary resolver flag is set it doesn't mean that a network dictionary is used. The network might not offer a dictionary for that chain so it will use the fallbackServiceUrl which would be a traditional dictionary.

Reply:
The fallback happens inside apollo-link, do you think we can alter the query at this level?

![image](https://github.com/subquery/subql/assets/40036742/db8be0ab-8d05-42db-9a5a-501d6d7dc924)

Reply:
Yes, I don't think its really possible for us to check